src/components/DiffView/index.tsx

```diff
@@ -48,12 +56,57 @@ export type Props = PublicProps & DefaultProps & RouterProps;
 export class DiffViewBase extends React.Component<Props> {
   static defaultProps: DefaultProps = {
     _document: document,
+    _getDiffAnchors: getDiffAnchors,
+    _getRelativeDiffAnchor: getRelativeDiffAnchor,
     _tokenize: tokenize,
     viewType: 'unified',
   };
 
   componentDidMount() {
-    const { _document, location } = this.props;
+    this.loadData();
+  }
+
+  componentDidUpdate() {
+    this.loadData();
+  }
+
+  loadData() {
+    const {
+      _document,
+      _getDiffAnchors,
+      _getRelativeDiffAnchor,
+      diff,
+      history,
+      location,
+    } = this.props;
+
+    if (diff) {
+      const queryParams = queryString.parse(location.search);
+      const { showDiff } = queryParams;
```

Comment:
`showDiff` makes me think that it's a boolean flag that will load some diff, but it's actually about revealing the anchor of the first diff of the next file/last diff of the previous file.

Could we find a more accurate name for this query parameter? `scrollPosition` maybe? In the future, we might want to navigate between comments, so we could have `firstDiff`, `lastDiff`, and later: `firstComment`, 	`lastComment`. I don't know, what do you think?

Reply:
Good idea. I replaced the URL parameter with `scrollTo`, and replaced the `DiffPosition` `enum` with `ScrollTarget`.